Detectors/FIT/FT0/reconstruction/src/CollisionTimeRecoTask.cxx

```diff
@@ -41,15 +41,16 @@ o2::ft0::RecPoints CollisionTimeRecoTask::process(o2::ft0::Digit const& bcd,
 
   Float_t sideAtime = 0, sideCtime = 0;
 
-  auto timeStamp = o2::InteractionRecord::bc2ns(bcd.mIntRecord.bc, bcd.mIntRecord.orbit);
+  // auto timeStamp = o2::InteractionRecord::bc2ns(bcd.mIntRecord.bc, bcd.mIntRecord.orbit);
 
-  LOG(DEBUG) << " event time " << timeStamp << " orbit " << bcd.mIntRecord.orbit << " bc " << bcd.mIntRecord.bc;
+  // LOG(DEBUG) << " event time " << timeStamp << " orbit " << bcd.mIntRecord.orbit << " bc " << bcd.mIntRecord.bc;
 
   int nch = inChData.size();
   const auto parInv = DigitizationParameters::Instance().mMV_2_NchannelsInverse;
   for (int ich = 0; ich < nch; ich++) {
+    int offsetChannel = mCalibOffset->mTimeOffsets.at(inChData[ich].ChId);
```

Comment:
`mTimeOffsets.at()` access is slower than `mTimeOffsets[]`, since the channel ID is well defined, use `[]`.